InputDate/index.jsx

```diff
@@ -1,31 +1,96 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import {
+  fontFamily,
+  fontSizeMini,
+} from '../style/font';
+import {
+  white,
+  curiousBlue,
+  outerSpace,
+} from '../style/color';
 import DayPicker from 'react-day-picker';
-import './style.css';
+import NavBar from './InputDateNavBar';
+import Weekday from './InputDateWeekday';
+import Caption from './InputDateCaption';
 import Text from '../Text';
+import moment from 'moment';
 
 /* eslint-disable react/prop-types */
+const datepickerStyles = {
+  display: 'flex',
+  flexWrap: 'wrap',
+  justifyContent: 'center',
+  position: 'relative',
+  userSelect: 'none',
+  fontFamily: fontFamily,
+  minWidth: '336px',
+};
+
+const modifierStyles = {
+  cell: {
+    display: 'inline-block',
+    padding: '.6rem 0rem',
+    width: '3rem',
+    textAlign: 'center',
+    cursor: 'pointer',
+    verticalAlign: 'middle',
+    overflow: 'hidden',
+    color: 'black',
+    fontSize: fontSizeMini,
+    border: '0.5px solid #eff1f1',
+    outline: '0.5px solid #eff1f1',
+    boxSizing: 'border-box',
+  },
+  today: {
+    color: curiousBlue,
+  },
+  outside: {
+    color: '#dce0e0',
+    cursor: 'default',
+  },
+  disabled: {
+    color: '#dce0e0',
+    cursor: 'default',
+    backgroundColor: '#f8f8f8',
+  },
+  selected: {
+    color: white,
+    backgroundColor: outerSpace,
+    border: 'none',
+    outline: 'none',
+  }
+};
+
+const modifiers = {
+  cell: { daysOfWeek: [0, 1, 2, 3, 4, 5, 6] },
+};
 
 const renderError = ({ error, touched }) => (
   error && touched ? (
     <div
       style={{
         marginTop: '1rem',
         textAlign: 'center',
+        width: '100%',
       }}
     >
       <Text color={'torchRed'}>{ error }</Text>
     </div>
   ) : null
 );
 
-const disabledDays = ({ disableBefore, submitting }) => {
+const disabledDays = ({ disableBefore, submitting }, initialMonth) => {
   if (submitting) {
     return () => true;
   } else if (disableBefore) {
     return {
       before: new Date(disableBefore.year, disableBefore.month, disableBefore.day),
     };
+  } else {
```

Comment:
Since all the conditionals return, the else an be removed

```
if (/* stuff */) {
  return {}
} else if () {
  return {}
}
return {
  before: initialMonth,
}
```